db/seeds.rb

```diff
@@ -7,4 +7,4 @@
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 #
 # Remember to ensure that the commands in this file are idempotent!
-Instance.find_or_create_by(name: 'Default', host: '*')
+Instance.find_or_create_by(name: 'Default', host: 'coursemology.org')
```

Comment:
@lowjoel  I think we need to validate the format of hostname ?   but here '*' is not a valid hostname, so I changed it.